src/devices/Hcsr04/Hcsr04.cs

```diff
@@ -40,32 +42,31 @@ public Sonar(int triggerPin, int echoPin)
         /// <summary>
         /// Gets the current distance in cm.
         /// </summary>
-        public double GetDistance()
+        private double GetDistance()
         {
             _timer.Reset();
 
             // Trigger input for 10uS to start ranging
             // ref https://components101.com/sites/default/files/component_datasheet/HCSR04%20Datasheet.pdf
+            _controller.Write(_trigger, PinValue.Low);
```

Comment:
why is Low needed here? It should be low already after previous measuring

Reply:
Start measuring the trigger pin that need to be written LOW -> HIGH -> LOW according to the datasheet. We don't know what the user will do after a measurement, so I think we should follow the datasheet.

Reply:
@ZhangGaoxing but we do control pin in this class and assume noone else is writing to it in here and it will be low already - I don't mind setting it explicitly but seems redundant to me since it's already at low